lib/gruntTasks.js

```diff
@@ -10,7 +10,7 @@ var sailsDbMigrate = require('./sailsDbMigrate');
  * @param command The db-migrate command to run.
  * @returns Arguments array for the db-migrate command.
  */
-function buildDbMigrateArgs(grunt, command) {
+function buildDbMigrateArgs(grunt, sails, command) {
```

Comment:
With the change recommended below, `sails` would change to `config`. Please add it to the JsDoc.